src/entities/communitySbt.ts

```diff
@@ -615,7 +645,7 @@ class SBT {
                 }
             }
         `;
-        const client = getApolloClient(this.badgesSubgraphUrl ?? "");
+        const client = getApolloClient(selectedBadgesSubgraphUrl ?? "");
```

Comment:
This is how we need to pass the fields from the UI. ``process.env.URL || ""`` and this is of type ``string`` not ``string?``